### PR TITLE
Quick module search bar with CTRL F

### PR DESCRIPTION
**Pull request by robinschmid**
CTRL F or Command F opens a popup to search and open modules:

Exit by escape, choose by enter

image

image

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -235,6 +235,7 @@ dependencies {
     implementation 'com.google.code.gson:gson:2.8.9'
     implementation 'io.gsonfire:gson-fire:1.8.5'
     implementation "org.threeten:threetenbp:1.3.8"
+//    https://kordamp.org/ikonli/cheat-sheet-bootstrapicons.html
```
Comment (SteffenHeu) [inline build.gradle:238]: ?

File: src/main/java/io/github/mzmine/modules/batchmode/ModuleQuickSelectDialog.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    scene.getStylesheets().addAll(mainWindow.getScene().getStylesheets());
+    this.setScene(scene);
+
+    scene.setOnKeyPressed(event -> {
+      if (event.getCode() == KeyCode.ESCAPE) {
+        hide();
+        event.consume();
+      }
+    });
+
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/batchmode/ModuleQuickSelectDialog.java:61]: i think clicking anywhere outside the search pane should also close the pane

**Review by SteffenHeu — COMMENTED** [review]
Some comments that would be nice to add

**Comment by SteffenHeu** [comment]
style sheet does not change

File: build.gradle (hunk 1 above)
Comment (robinschmid) [reply build.gradle:238]: just a link to the icons that we use

File: src/main/java/io/github/mzmine/modules/batchmode/ModuleQuickSelectDialog.java (hunk 2 above)
Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/batchmode/ModuleQuickSelectDialog.java]: done via focusProperty

File: src/main/java/io/github/mzmine/modules/batchmode/BatchModuleTreePane.java [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+      var firstMatchingNode = originalRoot.expandAllMatches(predicate);
+      treeView.getSelectionModel().select(firstMatchingNode);
+    });
+
+    searchField.setOnKeyPressed(event -> {
+      if (event.getCode() == KeyCode.ENTER) {
+        event.consume();
+        addSelectedModule();
+      }
+    });
```
[the remark this answers is not in the document]
Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/batchmode/BatchModuleTreePane.java]: done. needed to add another keyHandler there

**Comment by robinschmid** [comment]
Style sheet change fixed. 
arrows work. 
all done from my side

**Pull request merged**